code/modules/mob/living/living_say.dm

```diff
@@ -528,6 +532,14 @@ GLOBAL_LIST_INIT(message_modes_stat_limits, list(
 
 	return NONE
 
+/**
+ * Updates the verb variables on the living.
+ * This is primarily a hook for inheritors,
+ * like human_say.dm's tongue-based verb updates.
+ */
+/mob/living/proc/update_verbs(input, list/message_mods = list())
```

Comment:
`update_verbs` is a bad name for this given the existence of, well, `/verb`s. 
Perhaps `update_say_verb`.

Reply:
Fair point, I chose such purely cause there are other `verb_X`s, and making this specifically for `verb_say` in name felt off.